src/apb/cli.py

```diff
@@ -212,6 +212,20 @@ def subcmd_push_parser(subcmd):
         help=u'Namespace to push to internal OpenShift registry',
         default=u'openshift'
     )
+    subcmd.add_argument(
+        '--registry-service-name',
+        action='store',
+        dest='reg_svc_name',
+        help=u'Namespace where internal registry service lives',
```

Comment:
help copy paste error from namespace.  Should probably be `Service name where...`